js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -49,22 +52,39 @@ describe("Attestation", () => {
     expect(view.getByText("Operator #00000000")).toBeInTheDocument();
   });
 
-  test("contains signature text box", () => {
-    const view = render(
-      <Attestation
-        badge="123"
-        onComplete={jest.fn()}
-        loading={false}
-        employees={EMPLOYEES}
-      />,
-    );
-    expect(view.getByRole("textbox")).toBeInTheDocument();
+  describe("signature text box", () => {
+    test("it's there", () => {
+      const view = render(
+        <Attestation
+          badge="123"
+          prefill={false}
+          onComplete={jest.fn()}
+          loading={false}
+          employees={EMPLOYEES}
+        />,
+      );
+      expect(view.getByRole("textbox")).toBeInTheDocument();
+    });
```

Comment:
**nitpick:** when `prefill` is `false` can you add an additional assertion that the field starts out unfilled?

Reply:
👍 2b62b02